jQuery/honeyHeadquarters.js

```diff
@@ -1,4 +1,27 @@
 // $.ajax('https://sheetsu.com/apis/v1.0/64b5c3f8').done(function(data) {
-$.ajax('http://localhost:3000/citizens').done(function(data) {
-  console.log(data);
+$.ajax('http://localhost:3000/citizens').done(function(citizens) {
+  function getUniqueValuesForField(field, citizens) {
+    const values = ['All'];
+    citizens.forEach((c) => {
+      let v = c[field];
+      if (!values.includes(v)) {
```

Comment:
I just want you to know I'm going to be as belligerent as possible throughout this whole process